lib/language_pack/helpers/bundler_wrapper.rb

```diff
@@ -37,7 +37,7 @@ class LanguagePack::Helpers::BundlerWrapper
 
   BLESSED_BUNDLER_VERSIONS = {}
   BLESSED_BUNDLER_VERSIONS["1"] = "1.17.3"
-  BLESSED_BUNDLER_VERSIONS["2"] = "2.1.4"
+  BLESSED_BUNDLER_VERSIONS["2"] = "2.2.11"
```

Comment:
@schneems are you sure you want everyone with Bundler 2.1.x in their lockfile to get Bundler 2.2.11? https://github.com/heroku/heroku-buildpack-ruby/issues/1126#issuecomment-786236309

Reply:
Yes, we only have a single "blessed" version for 1.x and 2.x Bundler versions. It's not just 2.1 its 2.x that will get this version. 

Heroku needs to have a tight coupling on the bundler version to be able to provide support for it. Otherwise, we end up playing edge-case whack-a-mole and general behavior is undefined.